.circleci/create_circleci_config.py

```diff
@@ -110,6 +110,7 @@ def __post_init__(self):
             print(f"Using {self.docker_image} docker image")
         if self.install_steps is None:
             self.install_steps = ["uv venv && uv pip install ."]
+        self.install_steps.append("uv venv && uv pip install git+https://github.com/ydshieh/pytest.git@8.3.5-ydshieh git+https://github.com/ydshieh/pluggy.git@1.5.0-ydshieh")
```

Comment:
this is just my own forks for these 2 libraries with some more information printed.